packages/experiments/src/components/Chiclet/Chiclet.test.tsx

```diff
@@ -86,4 +82,28 @@ describe('Class Change', () => {
 
     expect(tree).toMatchSnapshot();
   });
+
+  it('should have the correct styling', () => {
+    const chicletCardProps: IChicletCardProps = {
+      url: 'contoso.sharepoint.com',
+      title: 'My Daily Notes',
+      itemType: 'docx',
+      onClick: () => alert('test')
+    };
+
+    const Prev: React.FunctionComponent<{}> = props => {
+      return <img src="http://placehold.it/100x100" {...props} />;
+    };
+
+    const component = mount(<ChicletCard {...chicletCardProps} preview={Prev} />);
+    const root = component.getDOMNode() as HTMLElement;
+    const prev = root.getElementsByClassName('ms-ChicletCard-preview')[0];
+    const innerPrev = prev.firstChild as HTMLElement;
+
+    const style = getComputedStyle(innerPrev);
+
+    expect(style.height).toBe('126px');
+    expect(style.width).toBe('100%');
+    expect(style.objectFit).toBe('contain');
```

Comment:
A snapshot test might be better suited for testing this type of output. If you already have one then I don't think the unit test is necessary. It will likely be too much of a maintenance burden compared with a snapshot.